README.md

```diff
@@ -1,12 +1,80 @@
 # Jenkins Lockable Resources Plugin
 
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/lockable-resources.svg)](https://plugins.jenkins.io/lockable-resources)
+[![GitHub release](https://img.shields.io/github/release/jenkinsci/lockable-resources-plugin.svg?label=release)](https://github.com/jenkinsci/lockable-resources-plugin/releases/latest)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/lockable-resources.svg?color=blue)](https://plugins.jenkins.io/lockable-resources)
 [![Build Status](https://ci.jenkins.io/buildStatus/icon?job=Plugins%2Flockable-resources-plugin%2Fmaster)](https://ci.jenkins.io/job/Plugins/job/lockable-resources-plugin/job/master/)
 [![GitHub license](https://img.shields.io/github/license/jenkinsci/lockable-resources-plugin.svg)](https://github.com/jenkinsci/lockable-resources-plugin/blob/master/LICENSE.txt)
 [![Maintenance](https://img.shields.io/maintenance/yes/2019.svg)]()
 
-This plugins allows to define "lockable resources" in the global configuration.
-These resources can then be "required" by jobs. If a job requires a resource
-which is already locked, it will be put in queue until the resource is released.
+This plugin allows defining lockable resources (such as printers, phones,
+computers, etc.) that can be used by builds. If a build requires a resource
+which is already locked, it will wait for the resource to be free.
+
+## Usage
+
+### Adding lockable resources
+
+1. In *Manage Jenkins* > *Configure System* go to **Lockable Resources
+   Manager**
+2. Select *Add Lockable Resource*
+
+Each lockable resource has the following properties:
+
+- **Name** - A name (not containing spaces!) for this particular resource, i.e.
+  `DK_Printers`
```

Comment:
```suggestion
  `DK_Printer_ColorA3_2342`
```
Change example resource name to distinguish between intended usages of `Name` and `Labels`.